python/source/flight.rst

```diff
@@ -752,4 +752,204 @@ Running the client script, you should see the server printing out information ab
 .. _IBM article: https://www.ibm.com/docs/en/arl/9.7?topic=certification-extracting-certificate-keys-from-pfx-file
 .. _Windows: https://docs.microsoft.com/en-us/dotnet/core/additional-tools/self-signed-certificates-guide
 .. _Arrow testing data repository: https://github.com/apache/arrow-testing/tree/master/data/flight
-.. _openssl: https://www.ibm.com/docs/en/api-connect/2018.x?topic=overview-generating-self-signed-certificate-using-openssl
\ No newline at end of file
+.. _openssl: https://www.ibm.com/docs/en/api-connect/2018.x?topic=overview-generating-self-signed-certificate-using-openssl
+
+Propagating OpenTelemetry Traces
+================================
+
+OpenTelemetry_ traces can be propagated between Flight clients and servers using
```

Comment:
Minor thing but I might start this paragraph with the problem being solved with the example. Something like:

> In order to correlate spans across a Flight client server, trace context must be passed between the two. There are two methods to do this: (1) middleware (2) manually via .... The example shown below uses a Flight middleware (hyperlink) which has the advantage of separating the implementation of the client or server from injecting and extracting trace context.